modules/social_features/social_user_export/social_user_export.module

```diff
@@ -85,6 +86,65 @@ function _social_user_export_get_view(array $query = [], $execute = TRUE) {
   return $view;
 }
 
+/**
+ * Returns first name of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return string
+ *   First name of the use.
+ *
+ * @throws InvalidPluginDefinitionException
+ */
+function social_user_export_first_name(UserInterface $account) {
+  $first_name = '';
+  /** @var \Drupal\profile\ProfileStorageInterface $storage */
+  // Check if entity type 'profile' exist
+  try {
+    $storage = \Drupal::entityTypeManager()->getStorage('profile');
+    // Checking if Storage is not empty.
+    if (!empty($storage)) {
+      if ($user_profile = $storage->loadByUser($account, 'profile', TRUE)) {
+        $first_name = $user_profile->get('field_profile_first_name')->value;
+      }
+    }
+  }
+  catch (InvalidPluginDefinitionException $e) {
+    $first_name = "No first Name is provided";
+  }
+  return $first_name;
+}
+
+/**
+ * Returns LastName of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return string
+ *   LastName of the user.
+ *
+ * @throws InvalidPluginDefinitionException
+ */
+function social_user_export_last_name(UserInterface $account) {
+  $last_name = '';
+  /** @var \Drupal\profile\ProfileStorageInterface $storage */
+  try {
+    $storage = \Drupal::entityTypeManager()->getStorage('profile');
+    // Checking if Storage is not empty.
+    if (!empty($storage)) {
+      if ($user_profile = $storage->loadByUser($account, 'profile', TRUE)) {
+        $last_name = $user_profile->get('field_profile_last_name')->value;
+      }
+    }
+  }
+  catch (InvalidPluginDefinitionException $e) {
+    $last_name = 'No last name is provided';
```

Comment:
Returning the empty $first_name here is probably better than a "magic constant".